src/scss/grommet-core/_objects.accordion-panel.scss

```diff
@@ -4,6 +4,10 @@
   header:focus {  // too generic, change
     outline: none;
   }
+
+  &.#{$grommet-namespace}list-item {
+    max-width: none;  // allow panel li to expand width fully
```

Comment:
I just changed the base li styling to not collide with list-item styling. So, I don't think you need this change anymore.